flink-dist/src/main/flink-bin/bin/flink-console.sh

```diff
@@ -58,7 +58,9 @@ esac
 
 FLINK_TM_CLASSPATH=`constructFlinkClassPath`
 
-log_setting=("-Dlog4j.configuration=file:${FLINK_CONF_DIR}/log4j-console.properties" "-Dlog4j.configurationFile=file:${FLINK_CONF_DIR}/log4j-console.properties" "-Dlogback.configurationFile=file:${FLINK_CONF_DIR}/logback-console.xml")
+log4j_config=`findLog4jConfiguration ${FLINK_CONF_DIR}`
+
+log_setting=("-Dlog4j.configuration=file:${log4j_config}" "-Dlog4j.configurationFile=file:${log4j_config}" "-Dlogback.configurationFile=file:${FLINK_CONF_DIR}/logback-console.xml")
```

Comment:
Reading on the JIRA ticket: seems the concern is user unable to override log settings original done `flink-console.sh` 
Given that, It seems odd to change only log4j but not logback.

Reply:
The goal was to make the scripts less strict when it comes to the file type (.properties vs .xml). This is more of a concern for log4j since we use a properties file, whereas xml appears to be the "preferred" way with log4j2.
I haven't heard similar problems existing with logback, but yes, ultimately it could be done for both frameworks.